tap_xero/client.py

```diff
@@ -178,9 +178,11 @@ def retry_after_wait_gen():
         yield math.floor(float(sleep_time_str))
 
 class XeroClient():
-    def __init__(self, config):
+    def __init__(self, config, config_path=None):
         self.session = requests.Session()
         self.user_agent = config.get("user_agent")
+        self.config = config
+        self.config_path = config_path
         self.tenant_id = None
```

Comment:
Should we just accept only `config_path` in `__init__()` and initialize `self.config` by reading the `config_path`? Any specific reason we are passing both?